index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header">
+        <p id="logo"> My Logo </p>
+        <img id="humburger-btn" src="images/icons8-hamburger-menu-64.png" alt="hamburger btn"/>
+    </header>
+    <main class="container">
+        <section class="first-section">
+            <h2>I’m Crispin<br/> Glad to see<br/> you!</h2>
+        </section>
+        <section class="second-section">
+            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        </section>
+        <section class="third-section">
```

Comment:
- [ ] I think that this is not the best use of the `section element`. I believe that the first section tag should contain the `<p>I’m a software developer!....</p>`, the `Let's connect`, and the social media icons. The section element represents a generic standalone section of the document. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info. **The single section should use flexbox to align its content.**